### DISCUSSION TITLE
Vertical padding option for webtoon mode?

### DISCUSSION THREAD

**Discussion by HubKing** [gotson/komga#1271] [Ideas]
The view mode I prefer is continuous vertical scrolling, just like web pages. For this, I chose "webtoon" mode. The problem is that there is no gap between pages, which makes it difficult to know where a page ends.  PDF viewers usually have some sort of gap between pages in continuous scroll mode, so having this option would be nice.

![image](https://github.com/gotson/komga/assets/10900489/96a5fe0c-b261-42cf-b619-c687f0aa76ae)

**Comment by gotson** [comment]
Duplicate of #1512
